documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -355,7 +355,7 @@ The documentation template below must be used as part of the API documentation i
 ```
 ### Authorization and authentication
 
-[Camara Security and Interoperability Profile](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-Security-Interoperability.md) provides details on how a client requests an access token.
+[Camara Security and Interoperability Profile](CAMARA-Security-Interoperability.md) provides details on how a client requests an access token.
```

Comment:
This is the one being changed in #186

Reply:
But what I mean is #186 and #188 are changing the same document line in different ways...

Reply:
Ah, yes ... that was the evil of search/replace. Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/188/commits/a4bcb0c67684e289010441cb7847362c35051589